src/modules/menu-category/menu-category.module.ts

```diff
@@ -8,5 +8,6 @@ import { MenuCategoryRepository } from './menu-category.respository';
   imports: [TypeOrmModule.forFeature([MenuCategoryRepository])],
   controllers: [MenuCategoryController],
   providers: [MenuCategoryService],
+  exports: [MenuCategoryService],
```

Comment:
why you exports this service?, you can add it in the menu-dishes module like the other modules under typeorm.forFeature function for example:
imports: [
    TypeOrmModule.forFeature([
      OrderDetailRepository,
      OrderRepository,
      UserRepository,
      MenuDishesRepository,
    ]),
  ],